scripts/install.js

```diff
@@ -92,37 +92,47 @@ function download(url, dest, cb) {
  */
 
 function checkAndDownloadBinary() {
-  if (sass.hasBinary(sass.getBinaryPath())) {
+  if (process.env.SKIP_SASS_BINARY_DOWNLOAD_FOR_CI) {
+    console.log('Skipping downloading binaries on CI builds');
     return;
   }
+  var binaryPath = sass.getBinaryPath();
 
-  mkdir(path.dirname(sass.getBinaryPath()), function(err) {
+  if (sass.hasBinary(binaryPath)) {
+    return;
+  }
+
+  mkdir(path.dirname(binaryPath), function(err) {
```

Comment:
Since it looks like fs.stat https://nodejs.org/api/fs.html#fs_fs_stat_path_callback isn't supposed to be called before doing the write, maybe try to create the cachePath here, then fallback to the vendor if it can't